### PR TITLE
Jump Game

### PR DESCRIPTION
**Pull request by rajbir21-droid**
The simple Jump game.

### CODE REVIEW

**Comment by rajbir21-droid** [comment]
Please review my PR

**Comment by chicken-biryani** [comment]
Hey @rajbir21-droid everything looks good. Can you add a small README.md file to your project... 

Contents of the README:

- Project overview.
- Use of the project.
- Stack of your project.
- How to set up / run your project.
- Screenshots of the pages / output.
A short description under each screenshot... Describing the page a little bit.


This would help visitors to get a good view of your project.

It seems tedious for now but it will be helpful.

**Comment by rajbir21-droid** [comment]
@chicken-biryani okay

**Comment by rajbir21-droid** [comment]
@chicken-biryani  updated the Readme.md . Please have a look

File: JAVASCRIPT GAME/Jump Game/index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    <p id="co">Press space to jump</p>
+    <div class="game">
+        <div id="character"></div>
+        <div id="block"></div>
+        <div id="block2"></div>
+    </div>
+    <p>Score: <span id="scoreSpan"></span></p>
+</body>
+<script src="script.js"></script>
+</html>
```
Comment (chicken-biryani) [inline JAVASCRIPT GAME/Jump Game/index.html:19]: Add a new line at the end of file.

Use https://codebeautify.org/htmlviewer to beautify your HTML part.

File: JAVASCRIPT GAME/Jump Game/Readme.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+* This web app allows you to play the jump game where you have to make box jump and safe himself from the small boxes coming in down and up direction . If the box touches the small box, the player lose. This is a fun game to play and a simple version of the one available in chrome when you are offline. Also, a good game to get started with Javascript.
+
+## Technologies used:
+* Html
+* Css
+* JS
+## How to set-up:
+* Clone this repo
+* Open it in any Text Editor 
+* Enjoy it !!!
```
Comment (chicken-biryani) [inline JAVASCRIPT GAME/Jump Game/Readme.md:13]: Use a markdown beautifier tool.

Here use this link https://codebeautify.org/markdown-formatter

**Review by Ayushparikh-code — APPROVED** [review]

**Pull request merged**